### ISSUE TITLE
Accessing colors globally

### ISSUE THREAD

**Issue by migcarva** [FlemingVincent/expo-supabase-starter#15]
I'm starting a new project and my implementation looks fairly close to yours. I tried using css variables but I was not able to make those globally available (didn't spent much time on this though). Do you have a solution for this? Let's say you need to pass a color to some Stack.screen

**Comment by FlemingVincent** [comment]
@migcarva This is project using CSS variables in combination with Tailwind CSS and [NativeWind](https://www.nativewind.dev/v4/overview). See [`global.css`](https://github.com/FlemingVincent/expo-supabase-starter/blob/main/global.css) and [`tailwind.config.js`](https://github.com/FlemingVincent/expo-supabase-starter/blob/main/tailwind.config.js) for the implementation.

**Comment by migcarva** [comment]
Thanks for replying @FlemingVincent 

Maybe I was a bit misleading with my question. Let me rephrase it: How do you access colors outside of the realm of CSS, classes? For instance, let's say you have this:

```
<Stack
      screenOptions={{
        headerStyle: {
          backgroundColor: '#f4511e', // how do you specify a color here
        },
        headerTintColor: '#fff',
        headerTitleStyle: {
          fontWeight: 'bold',
        },
      }}
    />
```

**Comment by FlemingVincent** [comment]
@migcarva Ah, yes, sorry I misunderstood your question. I went ahead and created and merged a PR for this. You can take a look at #17 which "Added ability to access colors globally. Updated expo-router."

Essentially, I created a constants file inside of the `lib` directory which contains a theme object consisting of light and dark mode colors that reflect the colors used in `global.css` and `tailwind.config.js`.
